### PR TITLE
[Opaque Pointer][LLVM][Test] Remove ifdef and port tests

### PR DESCRIPTION
**Pull request by jsji**
These tests were guarded within ifdef, so missed in last porting.

### CODE REVIEW

**Comment by jsji** [comment]
Ping @intel/dpcpp-tools-reviewers . Thanks.

**Review by bader — APPROVED** [review]

**Comment by jsji** [comment]
Thanks @bader ! @intel/llvm-gatekeepers  Can we get this merged. Thanks!

**Comment by bader** [comment]
@jsji, we need code owners' approval to merge this.

**Comment by jsji** [comment]
> @jsji, we need code owners' approval to merge this.

Ah... Sure. Did not notice that @intel/dpcpp-tools-reviewers haven't reviewed. Thanks for reminding.

File: llvm/test/Assembler/scalable-vector-struct.ll
Comment (asudarsa) [inline llvm/test/Assembler/scalable-vector-struct.ll:2]: I thought we remove the code inside ifndef. Can you please clarify?

Thanks

**Review by asudarsa — APPROVED** [review]
I think I understand the changes. Just requested a quick clarification. But the changes are fine.

Thanks

File: llvm/test/Assembler/scalable-vector-struct.ll
Comment (jsji) [reply llvm/test/Assembler/scalable-vector-struct.ll:2]: Yes, this was guarded because we were only testing typed pointer. This PR ported the testcase into opaque pointer, so we now can remove the guard to let it run.

**Comment by jsji** [comment]
@intel/llvm-gatekeepers Can we get this merged. Thanks!

**Pull request merged**
